core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -413,25 +413,38 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
    *
    * @param executorIds identifiers of executors to kill
    * @param replace whether to replace the killed executors with new ones
+   * @param force whether to kill busy executors (who are running tasks)
    * @return whether the kill request is acknowledged.
    */
-  final def killExecutors(executorIds: Seq[String], replace: Boolean): Boolean = synchronized {
+  final def killExecutors(executorIds: Seq[String],
+                          replace: Boolean,
+                          force: Boolean): Boolean = synchronized {
```

Comment:
nit: style for multiline method defs is each arg on its own line, double indented, so:

```scala
final def killExecutors(
    executorIds: Seq[String],
    replace: Boolean,
    force: Boolean): Boolean = synchronized {
```